bika/lims/browser/analysisrequest/workflow.py

```diff
@@ -202,25 +202,28 @@ def workflow_action_preserve(self):
             message = _("Saved items: {}".format(", ".join(transitioned)))
         self.redirect(message=message)
 
+    def requires_partitioning(self, brain_or_object):
+        """Returns whether the passed in object needs to be partitioned
+        """
+        obj = api.get_object(brain_or_object)
+        if not IAnalysisRequest.providedBy(obj):
+            return False
+        template = obj.getTemplate()
+        if not template or not template.getAutoPartition():
+            return False
+        partitions = template.getPartitions()
+        return partitions and len(partitions) > 0
```

Comment:
I think we have here with the widget always at least 1 partition, so I would remove that extra check